src/factors/Inertial/IMUDeltaFactor.jl

```diff
@@ -0,0 +1,467 @@
+using Manifolds
+using StaticArrays
+using Rotations
+using LinearAlgebra
+using DistributedFactorGraphs
+using Dates
+
+function TransformUtils.skew(v::SVector{3,T}) where T<:Real
+    return SMatrix{3,3,T}(
+            0,
+         v[3],
+        -v[2],
+        -v[3],  
+            0,  
+         v[1],
+         v[2],
+        -v[1],
+            0
+    )
+end
+
+struct IMUDeltaManifold <: AbstractManifold{ℝ} end
+
+# TODO maybe define the manifold like this
+# M = ProductManifold(TranslationGroup(3), TranslationGroup(3), SpecialOrthogonal(3), Euclidean(1))
+# M = ProductManifold(SpecialOrthogonal(3), TranslationGroup(3), TranslationGroup(3), Euclidean(1))
+# G = GroupManifold(M, MultiplicationOperation())
```

Comment:
I think doing `IMUDeltaGroup` like you did below is the better option. `ProductManifold` makes sense if you want to use the product metric but that's not what is used here.